src/ServiceTrait.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Prushak\Task7;
+
+use Prushak\Task7\Services\ConfigureService;
+
+trait ServiceTrait
+{
+    public function checkService($type)
+    {
+        switch ($type) {
+            case 'configure': return (new ConfigureService($type, rand(1, 10), rand(1, 10), rand(20, 100)));
```

Comment:
You need to use `break` at the end of each case block to prevent from the execution of the following case blocks

Reply:
return solves this problem. I had tasted.

Reply:
You are right. It can be :)